.eslintrc.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
     '@typescript-eslint/no-var-requires': 'warn',
     '@typescript-eslint/ban-ts-comment': 'warn',
     /* Temporary disabled rules for React development */
+    'react-hooks/rules-of-hooks': 'warn',
```

Comment:
What's the root cause of this change?

Reply:
@bjacquet would be the best person to answer this tbh

Reply:
Ahh so, I added this ignore rule because fixing the issue that it mentions would require changes to our implementation. Changes which I don't know how to make.

You can uncomment this to see what the issue is. It might be an easy fix for someone who knows react!!

Reply:
Have addressed the errors that appear after re-enabling this rule.